### PR TITLE
MAT-5610 Reformat the way valuesets and codes are specified to get QDM CQL to work work in MADiE

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR

Jira Ticket: [MAT-5610](https://jira.cms.gov/browse/MAT-5610)
(Optional) Related Tickets:
related PR: https://github.com/MeasureAuthoringTool/madie-util/pull/41

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Review by RohitKandimalla — DISMISSED** [review]

File: src/validations/editorValidation.ts [hunk 1]
```diff
@@ -29,9 +29,18 @@ export const useGetAllErrors = async (
...
+          isLoggedInUMLS.valueOf(),
+          cqlResult?.using?.name
+        ),
+
         TranslateCql(cql),
-        GetValueSetErrors(cqlResult.valueSets, isLoggedInUMLS.valueOf()),
+        GetValueSetErrors(
+          cqlResult.valueSets,
+          isLoggedInUMLS.valueOf(),
+          cqlResult?.using?.name
```
Comment (nmorasb) [inline src/validations/editorValidation.ts:42]: I see this value is what's getting passed down for `model`, which is then compared to "QDM". 
So this value would come out to be either `QDM`, `FHIR` or `QI-Core`? It would not include version numbers?
Also, I think Joe and Greg were hoping to avoid logic like "if QDM" and "if FHIR" in a bunch of places. I wonder if there's any way to accomplish this based on the other strategy being discussed?

Comment (adongare) [reply src/validations/editorValidation.ts:42]: Yes, `cqlResult?.using?.name` would be the model that is being used in cql(e.g. `using QDM version '5.6'`), it would not include version. As far as I know, I do not think value set validation would be based on the model & version. it would be only on the model. Except for getting the oids for value sets and code systems, I think our editor is pretty much model agnostic. So, not really sure if we should invest in separating the model-specific functionality for the editor. But I can update if we want to follow a different approach.

Comment (adongare) [reply src/validations/editorValidation.ts:42]: okay, as per our discussion, moved model-specific oid extraction to madie-utility(will be used in editor and patient spa). No more model checks in the editor. https://github.com/MeasureAuthoringTool/madie-util/pull/41

**Review by nmorasb — APPROVED** [review]
